scripts/src/report/templates/components/heading.html

```diff
@@ -17,4 +17,7 @@ <h1>{{ title }}</h1>
   <p>
     [analysis description + link to github repo]
   </p>
+  <p class="lead">
+    Database: {% if bold %}BOLD{% else %}BLAST{% endif %}
+  </p>
```

Comment:
Sorry, could you please move this above "analysis description"? Will look a bit better